br/backup-and-restore-tool.md

```diff
@@ -11,9 +11,18 @@ aliases: ['/docs/v3.1/br/backup-and-restore-tool/','/docs/v3.1/reference/tools/b
 ## Usage restrictions
 
 - BR only supports TiDB v3.1 and later versions.
-- Currently, TiDB does not support backing up and restoring partitioned tables.
-- Currently, you can perform restoration only on new clusters.
+- BR supports restore on clusters of different topologies. However, the online applications will be greatly impacted during the restore operation. It is recommended that you perform restore during the off-peak hours or use `rate-limit` to limit the rate.
 - It is recommended that you execute multiple backup operations serially. Otherwise, different backup operations might interfere with each other.
```

Comment:
```suggestion
- It is recommended that you execute multiple backup operations serially. Otherwise, different backup operations might interfere with each other.
- When BR restores data to the upstream cluster of Drainer, Drainer cannot replicate the restored data to the downstream.
```